### PR TITLE
OAK-9087: class / package names refactoring

### PR DESCRIPTION
**Pull request by fabriziofortino**

### CODE REVIEW

File: oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/ElasticIndexDefinition.java [hunk 1]
```diff
@@ -171,7 +171,7 @@ private static String getESSafeIndexName(String indexPath) {
                 .stream(PathUtils.elements(indexPath).spliterator(), false)
                 .limit(3) //Max 3 nodeNames including oak:index which is the immediate parent for any indexPath
                 .filter(p -> !"oak:index".equals(p))
-                .map(ElasticsearchIndexDefinition::getESSafeName)
+                .map(ElasticIndexDefinition::getESSafeName)
```
Comment (tihom88) [inline oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/ElasticIndexDefinition.java]: Should we also change ES with elastic.

File: oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/query/ElasticResultRowIterator.java [hunk 2]
```diff
@@ -153,14 +153,14 @@ private boolean loadDocs() {
 
         SearchHit lastDocToRecord = null;
         try {
-            ElasticsearchSearcher searcher = getCurrentSearcher(indexNode);
+            ElasticSearcher searcher = getCurrentSearcher(indexNode);
             QueryBuilder query = getESQuery(plan, planResult);
```
Comment (tihom88) [inline oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/query/ElasticResultRowIterator.java]: Same.

**Review by thomasmueller — APPROVED** [review]

**Comment by chibulcuteanu** [comment]
@fabriziofortino , shouldn't getters/setters also be changed here:
e.g
```
private ElasticConnection getElasticsearchConnection(Map<String, Object> contextConfig)
```
?

**Comment by fabriziofortino** [comment]
@chibulcuteanu Eagle eye :-) Fixed

**Review by chibulcuteanu — APPROVED** [review]

**Pull request closed without merging**